styles/style.css

```diff
@@ -1,31 +1,124 @@
-h1 {
-  text-align: center;
+@font-face {
+  font-family: "Roboto";
+  src: url("../fonts/Roboto-Regular.ttf");
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 24px;
+}
+*,
+*:before,
+*:after {
+  box-sizing: border-box;
+}
+.container {
+  width: 100%;
+  max-width: 1232px;
+  margin: 0 auto;
+}
+.header {
+  width: 100%;
+  background: url("../img/header_bg.jpg") no-repeat center / cover;
+}
+.header__inner {
+  display: flex;
+  justify-content: space-between;
+  padding: 50px 0;
+  align-items: center;
+}
+.nav {
+  display: flex;
+  font-size: 24px;
+  font-weight: 400;
+}
+.nav__links {
+  margin-left: 25px;
+  color: #fff;
+  text-decoration: none;
+}
+.nav__links:nth-child(3) {
+  margin-left: 50px;
 }
-hr {
-  border: 0;
-  background-color: #f5bd41;
-  width: 100px;
-  height: 4px;
+.nav__links:hover {
+  padding-bottom: 5px;
+  border-bottom: 4px solid #f5bd41;
+  text-transform: capitalize;
 }
-/* TEXT */
-.text {
+.intro {
   display: flex;
   justify-content: center;
-  margin-top: 20px;
-  margin-right: 40px;
+  height: 750px;
+  padding-top: 100px;
+  text-align: left;
 }
-.text_item {
-  width: 20%;
+.intro__inner {
+  width: 100%;
+  max-width: 600px;
+}
+.intro__title {
+  font-family: "Roboto", sans-serif;
+  font-size: 50px;
+  color: #fff;
+  font-weight: 500;
+  line-height: 59px;
```

Comment:
This is **a bad practice**, to use absolute values for line-height
For example if you need to change font-size for mobiles - `line-height: 24px` can look badly
Using of relative values will be better
```
line-height: 2.5;
line-height: 3em;
line-height: 150%;